monai/bundle/config_item.py

```diff
@@ -365,7 +364,15 @@ def evaluate(self, globals: Optional[Dict] = None, locals: Optional[Dict] = None
                 if k in globals_:
                     warnings.warn(f"the new global variable `{k}` conflicts with `self.globals`, override it.")
                 globals_[k] = v
-        return eval(value[len(self.prefix) :], globals_, locals)
+        if not run_debug:
+            return eval(value[len(self.prefix) :], globals_, locals)
+        warnings.warn(
+            f"\n\npdb: value={value}\n"
+            f"See also Debugger commands documentation: https://docs.python.org/3/library/pdb.html\n"
+        )
+        import pdb
```

Comment:
Should we import `pdb` at the beginning of the file?

Thanks.

Reply:
don't want to introduce this tool to the module, it's mostly not used, if we find it's frequently I'll update this.